src/patternfly/components/Tabs/examples/Tabs.md

```diff
@@ -199,18 +199,18 @@ import './Tabs.css'
 
 ### Using the nav element
 ```hbs
-{{#> tabs tabs--id="default-scroll-nav-example" tabs--type="nav" tabs--modifier="pf-m-scrollable" tabs--attribute='aria-label="Local"' tabs-link--isLink="true"}}
+{{#> tabs tabs--id="default-scroll-nav-example" tabs--type="nav" tabs--modifier="pf-m-scrollable" tabs--attribute='aria-label="Tabs nav"' tabs-link--isLink="true"}}
```

Comment:
We have this same duplication of the `aria-label` in the navigation component examples, too, and you can see that "Local" or "Global" is auto-generated as the `aria-label` in the `<Nav>` component depending on the type of nav used - https://github.com/patternfly/patternfly-react/blob/7bfaee093d3c85392a9d21a34ff661ffe29b0f47/packages/react-core/src/components/Nav/Nav.tsx#L162

I'm curious if that's an issue as well?

Reply:
On the react side, I overrode the default aria-label in the examples to avoid the duplicate aria label issue. I figure in product, the likelihood that they will run into duplicate labels is unlikely, and many products have to localize and override aria labels anyway.

Reply:
@nicolethoen can you clarify - I do see these duplicate aria label errors throw for "Global" and "Local", should we change here to fix those errors?  And if so do you have a recommendation, as it sounds like these specific labels have additional meaning